ci-operator/config/openshift/openstack-ironic/openshift-openstack-ironic-release-4.13.yaml

```diff
@@ -2,7 +2,7 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: ocp
-    tag: rhel-9-golang-1.19-openshift-4.13
+    tag: rhel-9-golang-1.20-openshift-4.14
```

Comment:
yep!

Reply:
this is intended, we're updating the root image because of issues with PBR that we also had in the past
we haven't been able to find the root cause, updating the image just works, so we suspect some weird caching mechanism
the base of the image is the same, tools are the same, and we only run tests on it, we don't publish anything

Reply:
ok, just saw the comment on slack, tks!